labelbox/client.py

```diff
@@ -95,6 +96,13 @@ def __init__(self,
             'X-User-Agent': f'python-sdk {SDK_VERSION}'
         }
         self._data_row_metadata_ontology = None
+        #
+        self.rest_endpoint = rest_endpoint
+        self.rest_endpoint_headers = {
+            "authorization": "Bearer %s" % self.api_key,
+            'X-User-Agent': 'python-sdk 0.0.0',
```

Comment:
Couple things here:
- Could we share the headers between graphql and rest?
- User agent is `0.0.0` here but using `SDK_VERSION` for graphql